packaging/rpm/scripts/after_install_host.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+echo "This software may collect information about you and your use of the software, and send that to Microsoft."
+echo "Please visit http://aka.ms/dotnet-cli-eula for more information."
+
+# Run 'dotnet new' as the user to trigger the first time experience to initialize the cache
+echo "Welcome to .NET Core!
```

Comment:
Compared to debian packages, RPMs are *non-interactive*. Its safe to assume a user may never see this, ever.